### PR TITLE
Add 20 new employers

### PR DESCRIPTION
**Pull request by ekumlin**
- Add employers:
    - Away
    - Boeing
    - BuzzFeed
    - Cisco
    - ClassPass
    - Compass
    - Dick's
    - Facebook
    - Getaround
    - La-Z-Boy
    - Lyft
    - Mondelez
    - Party City
    - SpaceX
    - Spirit AeroSystems
    - Spirit Airlines
    - StubHub
    - Toast
    - Uber
    - Wonderschool
- Update claims for Google, Instacart, and Yum Brands
- Fix a couple bugs

### CODE REVIEW

File: public/employers/boeing.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    type: publication
+    sources:
+      - source: Q13 Fox
+        link: https://web.archive.org/web/20200421043756/https://q13fox.com/2020/04/20/boeing-other-major-u-s-manufacturers-reopening-amid-fierce-political-heat/
+        date: 2020-04-20T18:38:18Z
+  - summary: Boeing announced that employees unable to work remotely would be paid for 10 days, receive usual benefits, be eligible for unemployment assistance, and be able to use accrued paid time off.
+    positivity: 1
+    type: publication
+    sources:
+      - source: Boeing News Releases
```
Comment (frosteeze) [inline public/employers/boeing.yml:44]: We should mention how Boeing is considering taking federal aid and weighing it against taking loans from investment banks. https://www.cnbc.com/2020/04/07/coronavirus-house-democrats-urge-boeing-to-take-federal-aid-as-plants-shut-down.html

Not sure the positivity of that however, if any.

File: public/employers/buzzfeed.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  state: NY
+  country: us
+  international: false
+  wiki: New York City
+employeesBefore:
+  type: approximately
+  upperBound: 1700
+  year: 2017
+  yearQuarter: Q4
+summary: BuzzFeed announced plans to avoid layoffs by significantly cutting executive salaries and giving gradual pay cuts to other salaried employees.
```
Comment (frosteeze) [inline public/employers/buzzfeed.yml:19]: They may have announced that, but they still laid off people. https://variety.com/2020/digital/news/buzzfeed-layoffs-twitter-am-to-dm-shutting-down-1234582127/

File: public/employers/facebook.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+name: Facebook
+ticker: FB
+wiki: Example
```
Comment (frosteeze) [inline public/employers/facebook.yml:3]: Should fix this

File: public/employers/boeing.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+name: The Boeing Company
+shortName: Boeing
+ticker: BA
+wiki: Example
```
Comment (frosteeze) [inline public/employers/boeing.yml:4]: Should fix this.

File: public/employers/facebook.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  international: true
+  wiki: Menlo Park, California
+employeesBefore:
+  source: Statista
+  sourceLink: https://www.statista.com/statistics/273563/number-of-facebook-employees/
+  type: exactly
+  upperBound: 44942
+  year: 2019
+  yearQuarter: Q4
+summary: Facebook announced that hourly workers, even if not working, would be paid through the pandemic, and provided financial assistance for remote workers get set up. However, contract employees were not eligible for additional paid leave or said financial assistance.
```
Comment (frosteeze) [inline public/employers/facebook.yml:21]: I think it'd be easier to just refer these workers as paid furloughed.

File: public/employers/party-city.yml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  - retail
+location:
+  city: Rockaway
+  state: NJ
+  country: us
+  international: true
+  wiki: Rockaway, New Jersey
+employeesBefore:
+  type: approximately
+  upperBound: 18300
```
Comment (frosteeze) [inline public/employers/party-city.yml:21]: Source?

File: public/employers/compass.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        date: 2020-03-24T18:00:00-04:00
+      - source: CNBC
+        link: https://web.archive.org/web/20200412193826/https://www.cnbc.com/2020/03/13/workforce-wire-coronavirus-heres-what-every-major-company-is-doing-about-the-pandemic.html
+  - summary: Compass offered laid-off employees health benefits, enhanced severance packages, company laptops, and training and networking tools to help them find employment.
+    positivity: 1
+    type: publication
+    sources:
+      - source: TechCrunch
+        link: https://web.archive.org/web/20200420161027/https://techcrunch.com/2020/03/23/with-the-real-estate-industry-facing-headwinds-softbank-backed-compass-lays-off-15-of-staff-after/
+        date: 2020-03-24T01:00:11Z
```
Comment (frosteeze) [inline public/employers/compass.yml:34]: Date is wrong, it's 23rd.

File: public/employers/classpass.yml [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+wiki: ClassPass
+image: classpass.svg#0055fe
+officialWebsite: https://classpass.com/about
+industries:
+  - healthAndFitness
+location:
+  city: New York City
+  state: NY
+  country: us
+  international: false
```
Comment (frosteeze) [inline public/employers/classpass.yml:12]: They are international.

File: public/employers/cisco.yml [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+  city: San Jose
+  state: CA
+  country: us
+  international: true
+  wiki: San Jose, California
+employeesBefore:
+  type: approximately
+  upperBound: 75900
+  year: 2019
+summary: Cisco, as one of the world's leading network companies, provides remote work access for employes, and assured them that their jobs are not in danger of being cut.
```
Comment (frosteeze) [inline public/employers/cisco.yml:20]: They are also helping struggling businesses by giving them a 90-day payment holiday.

File: public/employers/boeing.yml (hunk 1 above)
Comment (ekumlin) [reply public/employers/boeing.yml:44]: Good find! Neutral, I'd say.

File: public/employers/compass.yml (hunk 7 above)
Comment (ekumlin) [reply public/employers/compass.yml:34]: 23rd in their time zone. This is a UTC time.

**Pull request merged**